### PR TITLE
German user guide review

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ weight: 70
 
 Im Menü **Einstellungen** können Administratoren verschiedene Einstellungen des Kontos verwalten:
 
-- [Authentifizierungseinstellungen](#authentication) und [Single-Sign-On](#configuring-single-sign-on) konfigurieren.
+- [Authentifizierungseinstellungen](#authentication) und [Single-Sign-On](#konfigurieren-von-single-sign-on) konfigurieren.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:9]: We should not translate links. Better add a real anchor here.

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Im Feld **Gültigkeitsdauer des Passworts** können Sie die Gültigkeit von Benu
 
 >**Info:** Die Begrenzung der Passwort-Gültigkeitsdauer gilt für Benutzer mit der Rolle "devices". Sie verhindert, dass Gerätepasswörter ablaufen.
 
-Standardmäßig können Benutzer jedes Passwort verwenden, das 8 Zeichen oder mehr enthält. Wenn Sie **Verwenden von starken (grünen) Passwörtern erzwingen** auswählen, müssen die Benutzer starke Passwörter verwenden, wie unter [Erste Schritte > Benutzeroptionen und -einstellungen > So ändern Sie Ihr Passwort](/benutzerhandbuch/getting-started-de/#to-change-your-password) beschrieben.
+Standardmäßig können Benutzer jedes Passwort verwenden, das 8 Zeichen oder mehr enthält. Wenn Sie **Verwenden von starken (grünen) Passwörtern erzwingen** auswählen, müssen die Benutzer starke Passwörter verwenden, wie unter [Erste Schritte > Benutzeroptionen und -einstellungen > So ändern Sie Ihr Passwort](/benutzerhandbuch/getting-started-de/#so-ändern-sie-ihr-passwort) beschrieben.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:40]: see above

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 3]
```diff
@@ -271,7 +271,7 @@ Bei jeder Anmeldung des Benutzers weist die dynamische Rechtezuordnung standardm
...
 
-Wenn aktiviert, wird die dynamische Rechtezuordnung nur verwendet, wenn sich ein neuer Benutzer anmeldet, um die anfänglichen Rollen auszufüllen. Wenn in {{< product-c8y-iot >}} bereits ein Benutzer existiert, werden die Rollen weder überschrieben noch aktualisiert. Bei Aktivierung dieser Option können Administratoren auch die Rollen von SSO-Benutzern in der Benutzerverwaltung ändern. Nähere Informationen finden Sie unter [Administration > Verwalten von Berechtigungen](/benutzerhandbuch/administration-de/#assigning-global-roles) im *User Guide*.
+Wenn aktiviert, wird die dynamische Rechtezuordnung nur verwendet, wenn sich ein neuer Benutzer anmeldet, um die anfänglichen Rollen auszufüllen. Wenn in {{< product-c8y-iot >}} bereits ein Benutzer existiert, werden die Rollen weder überschrieben noch aktualisiert. Bei Aktivierung dieser Option können Administratoren auch die Rollen von SSO-Benutzern in der Benutzerverwaltung ändern. Nähere Informationen finden Sie unter [Administration > Verwalten von Berechtigungen](/benutzerhandbuch/administration-de/#so-weisen-sie-globale-rollen-aus-der-benutzerliste-zu) im *User Guide*.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:274]: see above

File: content/benutzerhandbuch/administration-de-bundle/managing-applications.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ title: Verwalten von Anwendungen
 weight: 30
 ---
 
-Die {{< product-c8y-iot >}}-Plattform unterscheidet zwischen Anwendungen und Microservices, siehe auch [Developing applications](concepts/applications/#overview) im *Concepts Guide*.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/managing-applications.md:7]: This link needs to be fixed in the English file as well.

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:9]: Fixed. Included anchor in the English and German guide.

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md (hunk 2 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:40]: Fixed

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md (hunk 3 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:274]: Fixed

File: content/benutzerhandbuch/administration-de-bundle/managing-applications.md (hunk 4 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/administration-de-bundle/managing-applications.md:7]: Already fixed

File: content/benutzerhandbuch/administration-de-bundle/managing-permissions.md [hunk 5]
```diff
@@ -253,12 +253,13 @@ Möglicherweise werden weitere Berechtigungen angezeigt, abhängig von den Funkt
 
 > **Wichtig:** Werden neue Funktionen mit neuen Berechtigungen zu {{< product-c8y-iot >}} hinzugefügt, so werden diese nicht automatisch zu bestehenden Rollen hinzugefügt. Sollten Sie feststellen, dass Sie eine kürzlich angekündigte Funktionalität nicht verwenden können, überprüfen Sie zunächst Ihre Berechtigungen.
 
-<a name="attach-global"></a>
+<a name="assigning-global-roles"></a>
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/managing-permissions.md:256]: If you change an existing anchor, you need to check through the entire documentation where this is used as link target. Therefore I would not touch existing anchors if not necessary.

File: content/benutzerhandbuch/administration-de-bundle/managing-permissions.md [hunk 6]
```diff
@@ -253,12 +253,13 @@ Möglicherweise werden weitere Berechtigungen angezeigt, abhängig von den Funkt
...
 #### Zuweisen von globalen Rollen
 
 Sie können Benutzern globale Rollen entweder direkt in der Benutzerliste oder auf der entsprechenden Benutzerseite zuweisen.
 
-> **Wichtig:** Standardmäßig ist es nicht möglich, die (bei der SSO-Anmeldung automatisch erstellten) Rollen von SSO-Benutzern zu ändern, da diese durch die dynamische Rechtezuordnung überschrieben würden. Dieses Verhalten kann jedoch geändert werden. Weitere Informationen finden Sie unter [Administration > Änderung von Einstellungen](/benutzerhandbuch/administration-de/#custom-template) im *User Guide*.
+> **Wichtig:** Standardmäßig ist es nicht möglich, die (bei der SSO-Anmeldung automatisch erstellten) Rollen von SSO-Benutzern zu ändern, da diese durch die dynamische Rechtezuordnung überschrieben würden. Dieses Verhalten kann jedoch geändert werden. Weitere Informationen finden Sie unter [Administration > Änderung von Einstellungen](/benutzerhandbuch/administration-de/#benutzerdefinierte-vorlage) im *User Guide*.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/managing-permissions.md:261]: Link still shows a German translation.

File: content/benutzerhandbuch/cockpit-de-bundle/overview.md [hunk 7]
```diff
@@ -83,6 +83,6 @@ Die Daten, die auf der Startseite angezeigt werden, werden von allen Benutzern d
 
 Die Startseite kann nach Ihren eigenen Bedürfnissen bearbeitet und gestaltet werden. Sie können die angezeigten Widgets ändern oder entfernen oder neue Widgets hinzufügen.
 
-Details zum Bearbeiten von Dashboards finden Sie unter [Arbeiten mit Dashboards](#dashboards).
+Details zum Bearbeiten von Dashboards finden Sie unter [Verwenden von Dashboards](#dashboards).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/overview.md:86]: @MWindel This probably needs to be changed in the translators database as well.

File: content/benutzerhandbuch/device-management-de-bundle/monitoring-and-controlling-devices.md [hunk 8]
```diff
@@ -192,7 +192,7 @@ In jeder Zeile werden die folgenden Informationen für einen Alarm angezeigt:
 
 Klicken Sie auf den Pfeil rechts in einem Eintrag, um die Zeile auszuklappen und weitere Details zum Alarm anzuzeigen.
 
-* **Status**: Enthält weitere Informationen zum Alarmstatus und zeigt den Alarmtypen an. Die Typ-Information wird verwendet, um die Priorität von Alarmen zu konfigurieren, siehe [Administration > Priorisieren von Alarmen](/benutzerhandbuch/administration-de#reprio-alarms).
+* **Status**: Enthält weitere Informationen zum Alarmstatus und zeigt den Alarmtypen an. Die Typ-Information wird verwendet, um die Priorität von Alarmen zu konfigurieren, siehe [Administration > Anwenden von Geschäftsregeln > Alarmregeln](/benutzerhandbuch/administration-de/#reprio-alarms).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/device-management-de-bundle/monitoring-and-controlling-devices.md:195]: @JoshuaAdemola This probably also needs to be fixed in the English file?

File: content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ Um ein neues Template zu erstellen, klicken Sie rechts in der oberen Menüleiste
 
 Bei den SmartREST-Templates handelt es sich um eine Sammlung von Anfrage- und Antwort-Templates, die verwendet werden, um CSV-Daten und {{< product-c8y-iot >}} Rest API-Aufrufe zu konvertieren. Sie können SmartREST-Templates etwa verwenden, um einfach Geräte zur Plattform hinzuzufügen anstatt jedes Mal manuell eine Anfrage zu schreiben.
 
-Um die Geräteintegration zu vereinfachen, stellt {{< product-c8y-iot >}} auch statische Templates bereit, so dass Sie keine eigenen Templates erstellen müssen. Diese Templates konzentrieren sich jedoch nur auf häufig verwendete Nachrichten für die Geräteverwaltung. Weitere Informationen zu statischen Templates finden Sie im [Device SDK Guide](/reference/smartrest-two#mqtt-static-templates).
+Um die Geräteintegration zu vereinfachen, stellt {{< product-c8y-iot >}} auch statische Templates bereit, so dass Sie keine eigenen Templates erstellen müssen. Diese Templates konzentrieren sich jedoch nur auf häufig verwendete Nachrichten für die Geräteverwaltung. Weitere Informationen zu statischen Templates finden Sie im [Reference Guide](/reference/smartrest-two#mqtt-static-templates).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md:17]: Same here, probably also needs to be fixed in the English file.

File: content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md [hunk 10]
```diff
@@ -121,7 +121,7 @@ Ein Antwort-Template enthält alle erforderlichen Informationen, um Datenwerte a
 
 Die Antwort wird dem SmartREST-Template hinzugefügt.
 
-Weitere Informationen finden Sie unter [SmartREST > Templates](/reference/smartrest-one/#response-templates) im *Reference Guide*.
+Weitere Informationen finden Sie unter [SmartREST 1.0 > Templates > Response Templates](/reference/smartrest-one/#response-templates) im *Reference Guide*.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md:124]: Fix in the English file as well. 

Response Templates > Response templates

File: content/benutzerhandbuch/getting-started-de-bundle/gui-features.md [hunk 11]
```diff
@@ -64,7 +64,7 @@ Auf kleineren Bildschirmen weicht das Layout geringfügig ab. Der Navigator ist
 <img src="/images/benutzerhandbuch/getting-started/getting-started-application-switcher.png" alt="Application switcher" style="max-width: 100%">
 
 
-Der Application Switcher zeigt alle {{< product-c8y-iot >}}-Anwendungen an, auf die Sie momentan Zugriff haben. Dies können [abonnierte Anwendungen](/benutzerhandbuch/administration-de/#subscribed-applications) oder [benutzerdefinierte Anwendungen](/benutzerhandbuch/administration-de#own-applications) sein. Klicken Sie einfach auf das Symbol für die gewünschte Anwendung, um sie als aktive Anwendung zu öffnen.
+Der Application Switcher zeigt alle {{< product-c8y-iot >}}-Anwendungen an, auf die Sie momentan Zugriff haben. Dies können [Abonnierte Anwendungen](/benutzerhandbuch/administration-de/#subscribed-applications) oder [Benutzerdefinierte Anwendungen](/benutzerhandbuch/administration-de#own-applications) sein. Klicken Sie einfach auf das Symbol für die gewünschte Anwendung, um sie als aktive Anwendung zu öffnen.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/getting-started-de-bundle/gui-features.md:67]: I would leave the small letters here as these are indirect inline links

File: content/users-guide/administration-bundle/managing-permissions.md [hunk 12]
```diff
@@ -263,7 +263,7 @@ There may be additional permissions visible depending on the features in your su
 
 > **Important:** When new features with new permissions are added to {{< product-c8y-iot >}}, these are not automatically added to existing roles. If you notice that you cannot use a new feature that was recently announced, check your permissions.
 
-<a name="attach-global"></a>
+<a name="assigning-global-roles"></a>
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-permissions.md:266]: @JoshuaAdemola Did you search for this link throughout the documentation?

File: content/benutzerhandbuch/device-management-de-bundle/monitoring-and-controlling-devices.md (hunk 8 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/device-management-de-bundle/monitoring-and-controlling-devices.md:195]: Fixed

File: content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md (hunk 9 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md:17]: Fixed

File: content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md (hunk 10 above)
Comment (JoshuaAdemola) [reply content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md:124]: Fixed

File: content/users-guide/administration-bundle/managing-permissions.md (hunk 12 above)
Comment (JoshuaAdemola) [reply content/users-guide/administration-bundle/managing-permissions.md:266]: Fixed. Changed anchor to original version.

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**